t/Makefile.am

```diff
@@ -303,6 +303,7 @@ EXTRA_DIST= \
 	shell/output \
 	shell/initrc/tests \
 	flux-jobs/tests \
+	scripts/strerror_symbol \
```

Comment:
Should this commit just be squashed with the one that adds `strerror_symbol`?

Reply:
Yes, just did that and force-pushed.

Reply:
Shall I set MWP on this?

Reply:
Sure!